indico/modules/events/persons/forms.py

```diff
@@ -38,3 +38,9 @@ def __init__(self, *args, **kwargs):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class ManagePersonListsForm(IndicoForm):
+    allow_custom_persons = BooleanField(_('Submitters can enter persons manually'), widget=SwitchWidget(),
```

Comment:
Should we negate this ("Disallow manually entering...")? This is (outside JACoW) the much less common option...